prober/derp.go

```diff
@@ -577,6 +580,196 @@ func runDerpProbeNodePair(ctx context.Context, from, to *tailcfg.DERPNode, fromc
 	return nil
 }
 
+// derpProbeBandwidthMu ensures that derpProbeBandwidthTUN doesn't run concurrently.
+// This is necessary to avoid conflicts trying to create the TUN device.
+var derpProbeBandwidthMu sync.Mutex
```

Comment:
This is specific to `derpProbeBandwidthTUN()`, and should be named to reflect that, and/or scoped to that function, especially if we support both methods (TUN and non-TUN) for now.

Reply:
Hehe, I had originally named it `derpProbeBandwidthTUNMu` and it just looked so ugly I dropped the `TUN`. You're right though.